### PR TITLE
Add env for sensitive logging

### PR DESCRIPTION
**Pull request by amshamah419**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/28654

## Description
Modified the ApiException class to only return headers when the `DEMISTO_SENSITIVE_LOGGING` environment variable is set to true.

## Must have
- [ ] Unit Test or Example Code
- [ ] Changelog entry

### CODE REVIEW

**Review by glicht — CHANGES_REQUESTED** [review]
Let's change `DEMISTO_SENSITIVE_LOGGING` to `DEMISTO_EXCEPTION_HEADER_LOGGING`. And log only if it is set to one of: true/yes/1.

**Comment by amshamah419** [comment]
@glicht Should be good to go now

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 [1]: https://pypi.org/project/demisto-py/#history
 
+## 2.0.20
+* Prevent headers from being logged when `DEMISTO_EXCEPTION_HEADER_LOGGING` environment variable is set to false.
```
Comment (glicht) [inline CHANGELOG.md:8]: Need to update this.

Comment (glicht) [inline CHANGELOG.md:8]:
```suggestion
* Log only headers in exceptions when `DEMISTO_EXCEPTION_HEADER_LOGGING` environment variable is set to true. This protects against possible sensitive data being logged in exceptions.
```

**Review by glicht — APPROVED** [review]

**Pull request merged**